### ISSUE TITLE
Redirect request: Disability comp migration

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#98843]
## Description
This ticket tracks redirects that will happen during the disability compensation migration (DCM). We're continuously updating this ticket to capture new redirects that need to be implemented.

### Batch 1
- Redirect 1:
  - Legacy URL: https://benefits.va.gov/compensation/claimexam.asp
  - Modern URL: https://www.va.gov/disability/va-claim-exam/
- Redirect 2:
  - Legacy URL: https://benefits.va.gov/compensation/dbq_disabilityexams.asp
  - Modern URL: https://www.va.gov/disability/how-to-file-claim/evidence-needed/
- Redirect 3:
  - Legacy URL: https://benefits.va.gov/compensation/docs/AO_C123_AFSpecialityCodesUnits.pdf
  - Modern URL: https://www.va.gov/resources/c-123-aircraft-agent-orange-exposure-and-disability-compensation/
 - Redirect 4:
   - Legacy URL: https://www.benefits.va.gov/compensation/mailingaddresses.asp
   - Modern URL: https://www.va.gov/disability/how-to-file-claim/
 - Redirect 5:
   - Legacy URL: http://www.benefits.va.gov/compensation/claims-postservice-pow.asp
   - Modern URL: https://www.va.gov/disability/eligibility/former-pows/ 
 - Redirect 6: 
   - Legacy URL: http://www.benefits.va.gov/compensation/claims-postservice-agent_orange-diabetes.asp
   - Modern URL: FILL IN ONCE NEW Q&A IS LIVE
 - Redirect 7: 
   - Legacy URL: http://www.benefits.va.gov/compensation/notice.asp
   - Modern URL: https://www.va.gov/disability/how-to-file-claim/evidence-needed/
 - Redirect 8: 
   - Legacy URL: 
   - Modern URL: 
 - Redirect 9:
   - Legacy URL: 
   - Modern URL:
 - Redirect 10:
   - Legacy URL:
   - Modern URL:
## Tasks
- [ ] IA: Create redirect ticket for batch 1

## Criteria
- [ ] All redirects have been implemented for the disability compensation migration

**Comment by kristinoletmuskat** [comment]
@aprocik1 just double checking (I think we had this convo in slack) -- you are going to let us know when these are ready to go, and we'll make a redirect ticket for PW. Does that sound right?

**Comment by aprocik1** [comment]
Yup, that's right. You don't need to take any action on these redirects right now.

**Comment by mnorthuis** [comment]
Closing this issue. Will open official redirect ticket when this work is done.
